common/src/main/java/com/wynntils/features/user/chat/ChatTimestampFeature.java

```diff
@@ -18,6 +21,8 @@
 import net.minecraftforge.eventbus.api.EventPriority;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
+@StartDisabled
```

Comment:
Hidden change (or I failed to read something)? Anyways, iirc legacy has this disabled by default too

Reply:
Good catch! I did not intend to sneak this in; I changed it when I was processing that feature and thought to myself "remember to put this in a separate change", and then promptly forgot about it.

I still think it should be done, but I'll do it separately.